ionic/__main__.py

```diff
@@ -788,6 +495,124 @@ async def on_lb_start(event: lb.LightbulbStartedEvent):
     )
 
 
+@bot.command()
+@lb.option("nojoy", "Ignores :rofl: and :joy: reactions",bool, default=True)
+@lb.option(
+    "hours",
+    "How far back should I pull messages from (int)",
+    int,
+    default=6,
+    max_value=24,
+    min_value=1,
+)
+@lb.option("channel", "Target channel to search", h.TextableGuildChannel, default=None)
+@lb.command("reactionrank", "List top 5 reactions in specified channel")
+@lb.implements(lb.SlashCommand)
+async def ReactionRank(ctx: lb.Context) -> None:
```

Comment:
[PEP 0008](https://peps.python.org/pep-0008/#function-and-variable-names) conventions typically have functions named "lowercase, with words separated by underscores".

Would you be happy to change this accordingly? @cchollar

Reply:
Sure thing, I'm not entirely sure why I didn't use PEP 0008 conventions in the first place,